From 6764c7ec73a84f33e9bdb5b3599e5147ca84355d Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Mon, 6 Nov 2017 15:35:54 +0200
Subject: [PATCH 2728/5242] MLK-16742-1: dts: arm64: imx8qxp: Set proper
 compatible string

commit  2b3955a39c2ad2fe741ed4fc2f1f00ec2ff987ec from
https://source.codeaurora.org/external/imx/linux-imx.git

Set proper compatible string in order to trigger
proper handling of constrain_period_size.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 9ae13b4..33d5e46 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -2063,7 +2063,7 @@
 	};
 
 	esai0: esai@59010000 {
-		compatible = "fsl,imx6ull-esai";
+		compatible = "fsl,imx8qm-esai";
 		reg = <0x0 0x59010000 0x0 0x10000>;
 		interrupts = <GIC_SPI 409 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8QXP_AUD_ESAI_0_IPG>,
-- 
1.7.9.5

